lib/hackney/pdf/get_templates.rb

```diff
@@ -1,12 +1,13 @@
 module Hackney
   module PDF
     class GetTemplates
```

Comment:
I feel that this is now a different use case e.g.

```suggestion
     class GetTemplatesForUser
```